### DISCUSSION TITLE
Welcome to react-native-otp-entry Discussions!

### DISCUSSION THREAD

**Discussion by anday013** [anday013/react-native-otp-entry#46] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ser-emejia** [comment]
Hello!, I have a question, Is this library support the automatically fill in SMS passcodes on iPhone? Thanks :)

**Reply by anday013** [reply]
Yes, it does :)

![IMG_4204](https://github.com/anday013/react-native-otp-entry/assets/48630069/29ada43b-dceb-4596-ba3b-ab09494dbb69)

**Reply by patphantuan251** [reply]
> Yes, it does :)
> 
> ![IMG_4204](https://private-user-images.githubusercontent.com/48630069/321723468-29ada43b-dceb-4596-ba3b-ab09494dbb69.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg4OTY2MzYsIm5iZiI6MTcyODg5NjMzNiwicGF0aCI6Ii80ODYzMDA2OS8zMjE3MjM0NjgtMjlhZGE0M2ItZGNlYi00NTk2LWJhM2ItYWIwOTQ5NGRiYjY5LlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE0VDA4NTg1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ4NWZlOTRmNTNkZjg3MDdlMzI4MjY3YmU4Yzc0OGRlMjk0ZWIwNWNkZTY0YzhmYWI2NmI5ZDBhMjE3MTY2NjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.u44O1-eUt_OxPKKVjiCGb1oNqYpXfIRJhBdVsOYM9iE)

how to do like this? please guide me

**Reply by anday013** [reply]
@patphantuan251 you do not need to do anything specific, it works out-of-box

**Reply by kbrandwijk** [reply]
When I try this, nothing happens when I tap the code above the keyboard. It's not filled in

**Reply by anday013** [reply]
@kbrandwijk are you on the latest version of the lib?

**Comment by nkuryoc** [comment]
I want to use my keyboard to fill in the code, is there a way to disable the generic soft input...?
